lib/sanford/server.rb

```diff
@@ -259,6 +261,8 @@ class Configuration
 
       def initialize(values = nil)
         super(values)
+        self.ip   = !ENV['SANFORD_IP'].to_s.empty?   ? ENV['SANFORD_IP']   : '0.0.0.0'
+        self.port = !ENV['SANFORD_PORT'].to_s.empty? ? ENV['SANFORD_PORT'] : nil
```

Comment:
@jcredding maybe change to setting a local var so you don't have to repeat the ENV var name?

```ruby
self.ip   = !(v = ENV['SANFORD_IP'].to_s).empty?   ? v : '0.0.0.0'
self.port = !(v = ENV['SANFORD_PORT'].to_s).empty? ? v : nil
```